flink-connector/README.md

```diff
@@ -1,20 +1,31 @@
 # Apache Flink Google Cloud Pub/Sub Connector
```

Comment:
i guess this technically isn't the "Apache" flink google cloud pub/sub connector

Reply:
I can see how this name is a bit confusing. It's supposed to be interpretted as a "Google Cloud Pub/Sub Connector" for "Apache Flink" instead of a "Flink Google Cloud Pub/Sub Connector" maintained by "Apache". Hopefully the comment below disambiguates the meaning. I'll look into changing the name if this becomes a common source of confusion.